cfme/tests/cli/test_appliance_log_rotate.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+
+import dateutil
+
+
+def advance_appliance_date_by_day(appliance):
+    txt_date = appliance.ssh_client.run_command('date --rfc-3339=ns').output
```

Comment:
Could you add a doc block here

Reply:
Well the name of this function is saying it all so I didn't think it was necessary to document it.

Reply:
Well the name of this function is saying it all so I didn't think it was necessary to document it.

Reply:
OK, although I think the name says it all.

Reply:
OK, although I think the name says it all.

Reply:
OK, although I think the name says it all.

Reply:
OK, although I think the name says it all.